lambdas/get-addresses/src/lib/param-store-client.ts

```diff
@@ -1,3 +1,16 @@
 import { SSMClient } from "@aws-sdk/client-ssm";
+import { NodeHttpHandler } from "@smithy/node-http-handler";
+import { Agent } from "https";
 
-export const SsmClient = new SSMClient({ region: "eu-west-2" });
+const DEFAULT_CONNECTION_TIMEOUT = 30000;
+const DEFAULT_SOCKET_TIMEOUT = 30000;
+
+const requestHandler = new NodeHttpHandler({
+    httpsAgent: new Agent(),
+    connectionTimeout: DEFAULT_CONNECTION_TIMEOUT,
+    socketTimeout: DEFAULT_SOCKET_TIMEOUT,
+});
+export const SsmClient = new SSMClient({
```

Comment:
```suggestion

export const SsmClient = new SSMClient({
```
Add some space for readability